api-dummy/composer.json

```diff
@@ -0,0 +1,48 @@
+{
+    "type": "project",
+    "license": "MIT",
+    "require": {
+        "php": "^7.1.3",
+        "ext-iconv": "*",
+        "ext-xml": "*",
+        "symfony/config": "^4.0",
+        "symfony/console": "^4.0",
+        "symfony/dependency-injection": "^4.0",
+        "symfony/flex": "^1.0",
+        "symfony/framework-bundle": "^4.0",
+        "symfony/http-foundation": "^4.0",
+        "symfony/http-kernel": "^4.0",
+        "symfony/monolog-bundle": "^3.2",
+        "symfony/routing": "^4.0",
+        "symfony/yaml": "^4.0"
+    },
```

Comment:
I assume this is pure Symfony 4. Wonder about the size of `vendor/` and how it compares to the current Silex version it is substituting?